src/rpc/util.h

```diff
@@ -293,11 +293,11 @@ struct RPCResult {
         : RPCResult{cond, type, m_key_name, false, description, inner} {}
```

Comment:
here as well? See also fab680ab9061a2dde0c4882caf002b057c1db8c6 from https://github.com/bitcoin/bitcoin/pull/26706

Reply:
fab680ab9061a2dde0c4882caf002b057c1db8c6 has a wider effect as it adds `std::move` as well

Here, clang-tidy warnings/errors are fixed only. And changes are pretty focused and self-contained.

I'm not sure how to proceed further. Any suggestions?

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/26707#issuecomment-1385313594).

Reply:
If you rebase directly on fab92a5a5ab45227dbb0aa6d2cf4557b157b17e6, you can avoid changing the commit hash and (potentially) wasting review resources.

Reply:
> If you rebase directly on [fab92a5](https://github.com/bitcoin/bitcoin/commit/fab92a5a5ab45227dbb0aa6d2cf4557b157b17e6), you can avoid changing the commit hash and (potentially) wasting review resources.

Done.